integration_tests/test_suites/dagster-gcp-live-tests/integration_tests/test_compute_log_manager.py

```diff
@@ -0,0 +1,61 @@
+import subprocess
+from pathlib import Path
+from typing import Mapping
+
+import pytest
+from dagster import (
+    DagsterEventType,
+    DagsterInstance,
+    EventRecordsFilter,
+    _check as check,
+)
+from google.cloud import storage as gcs
+
+from .conftest import get_bucket_client, get_credentials  # noqa: TID252
+
+
+@pytest.mark.parametrize(
+    "dagster_yaml",
+    ["json-credentials.yaml"],
+    indirect=True,
+)
+def test_compute_log_manager(
+    dagster_dev: subprocess.Popen,
+    bucket_client: gcs.Bucket,
+    prefix_env: str,
+    credentials: Mapping,
+    dagster_yaml: Path,
+) -> None:
+    subprocess.run(
+        ["dagster", "asset", "materialize", "--select", "my_asset", "-m", "gcp_test_proj.defs"],
+        check=True,
+    )
+    logs_captured_data = check.not_none(
+        DagsterInstance.get()
+        .get_event_records(
+            EventRecordsFilter(
+                event_type=DagsterEventType.LOGS_CAPTURED,
+            )
+        )[0]
+        .event_log_entry.dagster_event
+    ).logs_captured_data
+
+    assert logs_captured_data.external_stderr_url
+    assert logs_captured_data.external_stdout_url
+    stderr = get_content_from_url(logs_captured_data.external_stderr_url)
+    stdout = get_content_from_url(logs_captured_data.external_stdout_url)
+    assert stdout.count("Printing without context") == 10
+    assert stderr.count("Logging using context") == 10
+
+
+def _parse_gcs_url_into_uri(url: str) -> str:
+    return url.removeprefix(
+        "https://console.cloud.google.com/storage/browser/_details/computelogmanager-tests/"
```

Comment:
the url format can't be directly downloaded; so we need to do some gross munging to pull out the downloadable log key.

We actually hard code a similar string in the compute log manager itself, we should turn that into a constant and use it in both places ideally.